CONTRIBUTING.md

```diff
@@ -430,14 +430,14 @@ DOCKER_HUB_REPO=vmware EXTRA_TAG= VERSION_TAG=latest make build-all
 Windows plugin needs to be built on a windows environment
 1. export WIN_VM1=[Valid Windows VM IP]
 2. make build-windows-plugin
-3. rename the build/windows/vdvs-bin.zip to /build/windows/docker-volume-vsphere_windows_<release_tag>.zip
+3. rename the build/windows/vdvs-bin.zip to /build/windows/vsphere-storage-for-docker_windows_<release_tag>.zip
 
 ### Upload files to bintray
 Two files need to be uploaded to bintray at https://bintray.com/vmware/vDVS/vDVS_Windows
```

Comment:
Please check if this is correct. Shall we still be using vDVS?

Reply:
good catch :) . .updated!